platform/broadcom/raw-image.mk

```diff
@@ -4,6 +4,7 @@ SONIC_RAW_IMAGE = sonic-broadcom.raw
 $(SONIC_RAW_IMAGE)_MACHINE = broadcom
 $(SONIC_RAW_IMAGE)_IMAGE_TYPE = raw
 $(SONIC_RAW_IMAGE)_INSTALLS += $(BRCM_OPENNSL_KERNEL)
+$(SONIC_ONE_ABOOT_IMAGE)_INSTALLS += $(SYSTEMD_SONIC_GENERATOR)
```

Comment:
this is in correct, please fix.

Reply:
you missed platform/broadcom/aboot-image.mk as well.

Reply:
Not sure which file you're referring to, I don't see broadcom/aboot-image.mk , did you mean one-aboot.mk?